tool/internal_pkg/pluginmode/thriftgo/plugin.go

```diff
@@ -107,6 +107,18 @@ func HandleRequest(req *plugin.Request) *plugin.Response {
 		files = append(files, fs...)
 	}
 
+	if conv.Config.TplDir != "" {
+		if len(conv.Services) == 0 {
+			return conv.failResp(errors.New("no service defined in the IDL"))
+		}
+		conv.Package.ServiceInfo = conv.Services[len(conv.Services)-1]
```

Comment:
This can only support single service scenario. Pls wait for my PR to be merged and changed this statement.